shell/packages/sandstorm-db/db.js

```diff
@@ -56,6 +55,11 @@ if (Meteor.isServer && process.env.LOG_MONGO_QUERIES) {
 //       verifiedEmail: Only provided by some services. Cannot be directly edited by the user.
 //       main: True if this is the user's main identity.
 //       noLogin: True if the user does not trust this identity for account authentication.
+//       devName: Only present when `service` is "dev". A string containing the name used to create
+//                this dev account.
+//       emailTokens: Only present when `service` is "emailToken". An array of objects of the form
```

Comment:
Note that this establishes a precedent that `identities` contains potentially-sensitive information (even though it's only a hash) next to information which is commonly publicly visible. The problem with this is that it could be tempting for someone to publish the whole identities array when they really only care about the public profile data.

If we're careful to provide common ways to access this data correctly then this could be avoided. But we should also consider whether the "profile" parts should be in a sub-object that can be more explicitly published.

Another possibility would be to keep the email tokens in the "services" object, since that's where OAuth tokens are currently stored and it's generally understood to contain secrets. But arguably some other fields of `identities` (e.g. `main` and `noLogin`) should not be public either so this doesn't totally solve the problem.